CompanyEmployees.Core.Domain/Repositories/IRepositoryManager.cs

```diff
@@ -5,5 +5,6 @@ namespace CompanyEmployees.Core.Domain.Repositories;
 public interface IRepositoryManager
 {
     ICompanyRepository Company { get; }
```

Comment:
Consider placing the new `IAttendanceRepository` property before `ICompanyRepository` to maintain alphabetical order, which helps improve readability and consistency, especially as more repositories are added over time.

**Suggested change**
```suggestion
public interface IRepositoryManager
{
    IAttendanceRepository Attendance { get; }
    ICompanyRepository Company { get; }
    Task SaveAsync(CancellationToken ct = default);
}
```